src/constants/index.ts

```diff
@@ -1 +1,2 @@
 export const CONNECT_ACCOUNT_LINK = '/oauth2/connect';
+export const PLUGIN_ID = 'mattermost-plugin-msteams-interop';
```

Comment:
put all such plugin-dependent(which can be configured on plugin or changed in future) variables in a `.env` file so that they can be changed whenever needed without any change in code

from `.env` file load/read all such values together in a separate constant file may be like `pluginConstants.ts` and from here export for further usage